arrows.lic

```diff
@@ -0,0 +1,88 @@
+custom_require.call(%w[common common-crafting])
+
+class Arrows
+  include DRC
+  include DRCC
+
+  def initialize
+    @settings = get_settings
+    @bag = @settings.crafting_container
+    @bag_items = @settings.crafting_items_in_container
+    @belt = @settings.engineering_belt
+
+    arg_definitions = [
+      [
+        { name: 'type', options: %w[couger-claw boar-tusk sabertooth angiswaerd hele'la basilisk elsralael soot-stained ice-adder jagged-horn drake-fang], variable: true, descriptions: "Which arrow type to create?" },
+        { name: 'lumber', regex: /\w+/i, variable: true, description: 'Type of lumber used.' },
+        { name: 'tools', options: %w[true false], variable: true, description: 'Get tools from clerk?'}
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+
+    @type = args.type
+    @lumber = args.lumber || 'maple'
+    tools = args.tools
+
+    get_tools if tools
+
+    check_flights
+    make_shafts
+    make_arrowheads
+    wait_for_script_to_complete('shape', ['stow', '5', "#{@type} arrows", 'shafts', 'arrows'])
+    store_tools if tools
+  end
+
+  def get_tools
+    wait_for_script_to_complete('clerk-tools', ['engineering', 'get'])
+  end
+
+  def store_tools
+    wait_for_script_to_complete('clerk-tools', ['engineering', 'store'])
+  end
+
+  def order_lumber
+    return unless bput('tap my lumber', 'You tap', 'What were') == 'What were'
```

Comment:
:bulb: `exists?('lumber')` ?

Reply:
I can change to that.

Reply:
reminder to include DRCI and common-items for `exists?`